README.md

```diff
@@ -102,6 +102,19 @@ public MainPage()
     };
 }
 ```
+#### Dispatcher Queue for WinUI
+When using the WinUI package, the Dispatcher given should be of type `DispatcherQueue`
```

Comment:
I think you could rephrase this to explain that `CoreDispatcher` doesn't exist on WinUI and `DisptacherQueue` is the equivalent so the implementation names are different.